### DISCUSSION TITLE
Hard to adjust theme

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#6678] [Ideas]
Yesterday on a call, a user was not sure how to change the app theme. When I showed them where it was, they were confused by the icons and tried to click along the slider. 

<img width="254" alt="CleanShot 2022-07-13 at 09 48 16@2x" src="https://user-images.githubusercontent.com/49767913/178692308-0d0702f1-fa7b-483e-8cb9-f183b6d1cd54.png">

**Comment by shogunpurple** [comment]
Converting to discussion as it's not a bug and the slider works as expected

**Comment by melohagan** [comment]
In my opinion allowing the user to click along the slider is intuitive.
